framework/src/org/checkerframework/framework/util/typeinference/constraint/AFReducingVisitor.java

```diff
@@ -585,4 +594,15 @@ public void visitWildcardAsSuperType(
         // in fact
         addInverseConstraint(subtype, supertype.getExtendsBound(), constraints);
     }
+
+    /**
+     * Checks that the wildcard is not an uninferred type argument. If it is, errorAbort will be
+     * called.
+     */
+    private void checkForUninferredTypes(AnnotatedWildcardType wildcardType) {
+        if (wildcardType.isUninferredTypeArgument()) {
+            ErrorReporter.errorAbort(
```

Comment:
Won't this now simply terminate in a different location with a different error message?
How does this make the test case pass? Or is the important fix the fix in InternalUtils and this change is to detect similar problem faster in the future?